frontend/src/ScoreboxRow.js

```diff
@@ -1,55 +1,56 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
 
 /**
  * Three colored boxes with numerical rating values, used in the course description box.
  */
 class ScoreboxRow extends Component {
   render() {
     function numOrNA(num) {
-        return isNaN(num) ? "N/A" : num.toFixed(1);
+      return isNaN(num) ? 'N/A' : num.toFixed(1)
     }
 
     function getColor(num) {
-        if (isNaN(num)) {
-            return "rating-good";
-        }
-
-        num = num.toFixed(1);
-
-        if (num < 2) {
-            return "rating-bad";
-        }
-
-        if (num < 3) {
-            return "rating-okay";
-        }
-
-        return "rating-good";
+      if (isNaN(num)) {
+        return 'rating-good'
+      }
+      num = num.toFixed(1)
+
+      if (num < 2) {
+        return 'rating-bad'
+      }
+      if (num < 3) {
+        return 'rating-okay'
+      }
+      return 'rating-good'
```

Comment:
could be nice to have this in a `helperFunctions.js` or `mapper.js` if there are a bunch of functions like this across the app. I think we as an org put too much logic in react components.

Reply:
I'll add a story to clubhouse for this, priority rn is just getting eslint setup and the style ironed out mostly